packages/remix-server-runtime/cookies.ts

```diff
@@ -60,24 +60,24 @@ export interface Cookie {
   parse(
     cookieHeader: string | null,
     options?: CookieParseOptions
-  ): Promise<any>;
+  ): Promise<Value | null | "">;
```

Comment:
Adding a literal type for `""` is necessary because the way `parse()` is written means that a cookie may be set but have an empty string value. 

If this is accurate, these types should be correct. 

If this is _not_ the case, it might be a good idea to remove the `return "";` from the `parse` function and let it only return `Value | null`.

Reply:
What are your thoughts on usage of `unknown` vs `any`?
- https://dmitripavlutin.com/typescript-unknown-vs-any/#:~:text=Conclusion,to%20perform%20operations%20on%20unknown%20